src/Text/Text.stories.tsx

```diff
@@ -13,31 +13,49 @@ type Story = StoryObj<typeof Text>;
 
 export const Base: Story = {
   args: {
-    children: 'Hello World',
-    variant: 'bodyMd',
+    children:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam feugiat, velit vel egestas rutrum, arcu ante suscipit mauris, eu eleifend mi ipsum et tellus.',
+    variant: 'body16',
     alignment: 'start',
     fontWeight: 'regular',
     as: 'p',
     color: 'black',
   },
 };
 
+export const Responsive: Story = {
+  args: {
+    ...Base.args,
+  },
+  render: ({ children }) => (
+    <Text
+      as="h1"
+      variant={{
```

Comment:
Could you please document this option too. Either via a code comment or in the MDX file. 

Also, it's worth pointing out that, folks don't have to specify each breakpoint. For example, in this case `xl` will use `body18`:

```js
  base: 'body16',
  sm: 'heading56',
  md: 'body18', 
```

Reply:
Yes totally. This need documentation before it goes out. I moved this PR to draft until this is done.